tests/baselines/reference/inlineJsxFactoryDeclarationsLocalTypes.errors.txt

```diff
@@ -3,7 +3,7 @@ tests/cases/conformance/jsx/inline/component.tsx(4,140): error TS2532: Object is
 tests/cases/conformance/jsx/inline/index.tsx(5,1): error TS2741: Property '__predomBrand' is missing in type 'import("tests/cases/conformance/jsx/inline/renderer").dom.JSX.Element' but required in type 'import("tests/cases/conformance/jsx/inline/renderer2").predom.JSX.Element'.
 tests/cases/conformance/jsx/inline/index.tsx(21,22): error TS2786: 'MySFC' cannot be used as a JSX component.
   Its return type 'import("tests/cases/conformance/jsx/inline/renderer2").predom.JSX.Element' is not a valid JSX element.
-    Property '__domBrand' is missing in type 'import("tests/cases/conformance/jsx/inline/renderer2").predom.JSX.Element' but required in type 'import("tests/cases/conformance/jsx/inline/renderer").dom.JSX.Element'.
+    The '__domBrand' attribute is missing, but is required for this 'MySFC' tag.

```

Comment:
Hm, this error seems.... bad. The new error gives no hints that it's not that you've messed up a property, but that you're outright passing a JSX element from the wrong JSX renderer!

Reply:
Auh, I have only used React in desktop with its default render version. I need some kind help and advice.... @DanielRosenwasser